CHANGELOG.rst

```diff
@@ -14,6 +14,7 @@ Changelog
 * Updated the minimum supported Rust version (MSRV) to 1.56.0, from 1.48.0.
 * Added support for the :class:`~cryptography.x509.OCSPAcceptableResponses`
   OCSP extension.
+* Restored X509 verification error codes to the OpenSSL bindings.
```

Comment:
Please remove this, we don't want to be documenting them here -- bindings exist to serve their users, we'll document the public API on the pyOpenSSL side.

Reply:
Done.